### ISSUE TITLE
Every Minute, replace a random users profile picture with a soyjak

### ISSUE THREAD

**Issue by Tim79000** [twitter/the-algorithm#2050]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by Ataraxia-Mechanica** [comment]
Yes, except they should make it so it's only the person I specifically not like gets the change.

**Comment by MadKamel** [comment]
Maybe there could be a feature where you can report users and they get a soyjak automatically generated for them (regardless of whether or not they actually did anything wrong). Exceptions could be users who Elon Musk himself determines to be based (as well as Elon himself).

**Comment by colleirose** [comment]
https://x.com/highprogressive/status/1806187418275479653
